src/khiva/matrixInternal.cpp

```diff
@@ -72,7 +72,7 @@ void InitProfileMemory(SCAMP::SCAMPArgs &args) {
             }
             break;
         }
-        //case SCAMP::PROFILE_TYPE_SUM_THRESH: {
+        // case SCAMP::PROFILE_TYPE_SUM_THRESH: {
```

Comment:
Shall this comment be here?, if it is and extension for the future no problem at all.